packages/core/src/cache/classes/ImageVolume.ts

```diff
@@ -181,19 +203,7 @@ export class ImageVolume implements IImageVolume {
 
   /** return true if it is a 4D volume or false if it is 3D volume */
   public isDynamicVolume(): boolean {
-    return false;
-  }
-
-  /**
-   * Return the scalar data for 3D volumes or the active scalar data
-   * (current time point) for 4D volumes
-   */
-  public getScalarData(): PixelDataTypedArray {
-    if (isTypedArray(this.scalarData)) {
-      return <PixelDataTypedArray>this.scalarData;
-    }
-
-    throw new Error('Unknown scalar data type');
+    return this.numTimePoints && this.numTimePoints > 1;
```

Comment:
return this.numTimePoints > 1;
is sufficient, as numTimePoints not defined or null or !this.numTimePoints all result in false for that > 1.